### DISCUSSION TITLE
Deployment error

### DISCUSSION THREAD

**Discussion by thkim0321** [alshedivat/al-folio#676] [Q&A] [answered]
Hi, thank you so much for the great source for the academic web page.

I am facing errors when I try to deploy my website to Github pages. 

- I am using v0.3.5
After forking it, rename the repository to <my-github-username>.github.io.
- In _config.yml, I set url to https://<your-github-username>.github.io and leave baseurl empty.
- It works fine locally.

My `Actions` page says there is an error `pages build and deployment`. It looks like something goes wrong in the process of `Build page with Jekyll`. I would appreciate it if I can get some hints to solve this problem. 

<img width="1073" alt="Screenshot 2022-05-20 at 16 53 59" src="https://user-images.githubusercontent.com/55601503/169555001-50958fb0-1a7c-408c-a468-beee790e5d4b.png">

<img width="1440" alt="Screenshot 2022-05-20 at 16 59 24" src="https://user-images.githubusercontent.com/55601503/169556040-7c4bd23a-c1d8-4311-872f-91cb1e8f7626.png">

<img width="699" alt="Screenshot 2022-05-20 at 16 59 55" src="https://user-images.githubusercontent.com/55601503/169556136-3eaa56b0-93cd-4a34-91f5-714408025827.png">

**Comment by thkim0321** [comment, marked as answer]
I have solved the problem by setting the branch to gh-pages from master (the fourth procedure of `To enable automatic deployment:`). I didn't do this procedure before because I thought the third procedure was not completed successfully due to the error sign in `Action` tab. 

Additionally, the first procedure was also not clear to me. It says `Click on Actions tab and Enable GitHub Actions`, but I don't see any `Enable GitHub Action` in my `Actions` tab.

But anyway, it worked!

**Reply by casualcomputer** [reply]
How did you find "Enable GitHub Actions" eventually ("gh-pages" is not an option)? You only said you couldn't follow the instruction, but you provided no details on a solution.
